docs/_slides/unaizorrilla.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: slide
+title: "Welcome to our slide deck!"
+---
+
+Greetings from unaizorrilla!
+
+![dinotocat](https://octodex.github.com/images/dinotocat.png)
```

Comment:
Hi @c-gomez-m what do you think about my image??

Reply:
```suggestion
![dinotocat](https://octodex.github.com/images/manufacturetocat.png)
```